src/dashboard/Data/Browser/DataBrowser.react.js

```diff
@@ -45,7 +45,7 @@ export default class DataBrowser extends React.Component {
       firstSelectedCell: null,
       selectedData: [],
       prevClassName: props.className,
-      panelWidth: 300,
+      panelWidth: 400,
```

Comment:
Why this change?

Reply:
Sorry, both of them were unintentional changes, i will revert them back in next commit.